utils.test.js

```diff
@@ -1421,21 +1522,23 @@ test("parsePkgLock v2", async () => {
     version: "2.2.1"
   });
   expect(deps[deps.length - 1].name).toEqual("rollup");
-
-  const pkgFilePath = new URL('test/data/package-json/v2/package-lock.json', import.meta.url).pathname;
-  expect(deps[deps.length - 1].evidence).toEqual({
-    identity: {
-      field: "purl",
-      confidence: 1,
-      methods: [
-        {
-          technique: "manifest-analysis",
-          confidence: 1,
-          value: pkgFilePath
-        }
-      ]
-    }
-  })
+  // pkgFilePath does not parse correctly on Windows.
+  if (process.env.OS !== "Windows_NT") {
+    const pkgFilePath = new URL('test/data/package-json/v2/package-lock.json', import.meta.url).pathname;
```

Comment:
Since this is used only for expect can we just hardcode the string instead of using URL class? That way we can remove the os check